### PR TITLE
Docker support

### PR DESCRIPTION
**Pull request by sardaukar**
With some support for Linux's `xdg-open` (https://portland.freedesktop.org/doc/xdg-open.html) too.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -59,7 +63,7 @@ rundecker: decker
 .PHONY: jsres
 js: jsres
 	@mkdir -p js/build/
-	@echo "VERSION=\"${VERSION}\"\n" > js/build/lilt.js
+	@echo "VERSION=\"${VERSION}\"" > js/build/lilt.js
```
Comment (sardaukar) [inline Makefile:66]: Not sure what this was, but made my build blow up.

**Comment by sardaukar** [comment]
Currently have my image live [here]

[here]: https://hub.docker.com/r/sardaukar/decker

**Comment by JohnEarnest** [comment]
What's the use-case for adding docker support?

Flagging out `open`/`xdg-open` is a good idea, but it's an unrelated change that should be in its own PR.

**Comment by JohnEarnest** [comment]
In this case, it seems like you're only building the JS version of Decker, which is little more than concatenating a few text files together. As it is, there are no libraries, packages, or environment configurations to "isolate" in a VM. Even running a local web server to "host" a local instance of web-decker is unnecessary, because web-decker consists of a single amalgamated file; you just open it in a web browser as a file, and it works.

I just don't understand the value-add here.

**Comment by sardaukar** [comment]
I guess not. I'll close the PR.

**Comment by gingerbeardman** [comment]
It's a shame to see this closed.

We have install & uninstall scripts, but no Docker (an arguably cleaner approach that avoids the need to use either install or uninstall).

**Pull request closed without merging**